test_scripts/Polices/appID_Management/TS01.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary: 
+--     [OnAppInterfaceUnregistered] "APP_UNAUTHORIZED" in case of failed nickname validation after updated policies
+-- Description: 
+--     In case the application with "appName" is successfully registered to SDL
+--     and the updated policies do not have this "appName" in this app's specific policies
+--     (= "nicknames" filed in "<appID>" section of "app_policies")
+--     SDL must send OnAppInterfaceUnregistered (APP_UNAUTHORIZED) to such application.
+-- 
+-- Test:
+--     1. Preconditions:
+--        - Update Policy Table by assigning for "0000001" application additional nickname = "App1"
+--        - Create new mobile session and register "app_new" with nickname = "App1"
+--        - Default policies will be used for "app_new"
+--     2. Steps:
+--        - Update Policy Table by adding "New Application" nickname for "app_new"
+--     3. Expected result:
+--        - Response OnAppInterfaceUnregistered has the following data: success = true, reason = "APP_UNAUTHORIZED"
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+  config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]  
+  local Common = require("user_modules/shared_testcases/Policy_app_ID_Management_Common")
+  local MobileSession = require("mobile_session")
+
+--[[ General Settings for configuration ]]
+  Test = require("connecttest") 
+  
+--[[ Preconditions ]]  
+  function Test:Pre_UpdatePolicy()
+    Common:UpdatePolicyTable(self, "files/jsons/Policies/appID_Management/ptu_1.json")
```

Comment:
@dboltovskyi It's not a good idea to replace the configuration file, because in case sdl_preloaded_pt.json is changed you should update also ptu_1.json.
You can check in /use_modules/shared_testcases/TestCasesForPolicyTable.lua usage of functions and if they are applicable to use or to create a new one: ex. testCasesForPolicyTable:createPolicyTableWithoutAPI, 

Important: If you need to update your file, please take last commit for https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/32